src/main/kotlin/net/atos/client/sd/model/templates/TemplatesDeclarations.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.sd.model.templates
+
+import nl.lifely.zac.util.NoArgConstructor
+
+@NoArgConstructor
+class TemplatesResponse {
```

Comment:
Can these not be Kotlin data classes? Those `lateinit vars` feel like a strange workaround that should not be needed?

Reply:
With data class the JSON deserialization fails. Tried annotating a secondary constructor but with no luck

Reply:
Ah ok. Even if you use all `var`'s instead of `val`'s (like we also do for `RESTZaak` etc currently)?